tests/scenario/test_machine_charm/test_relation_priority.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+import json
+from unittest.mock import patch
+
+import pytest
+from charms.grafana_agent.v0.cos_machine import COSMachineProvider
+from scenario import Relation, State
+from scenario import trigger as _trigger
```

Comment:
How are you able to use `trigger` below if you import it as `_trigger`?

Reply:
see line 17